lib/spree/conekta/credit_card.rb

```diff
@@ -31,6 +31,9 @@ def build_card(card)
         @id    = card['id']
         @brand = card['brand']
         @last4 = card['last4']
+        @exp_month = card['exp_month']
+        @exp_year  = card['exp_year']
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.